xbmc/ContextMenuItem.cpp

```diff
@@ -36,7 +34,10 @@ std::string CContextMenuItem::GetLabel() const
     return "";
 
   if (StringUtils::IsNaturalNumber(m_label))
-    return m_addon->GetString(boost::lexical_cast<int>(m_label.c_str()));
+  {
+    int number = std::stoi(m_label);
```

Comment:
I don't think this is available on Android. Jenkins will probably reveal whether I'm correct or not.